app/templates/screen/Presenter.kt.mustache

```diff
@@ -36,14 +36,14 @@ class {{screenUpperCamel}}Presenter(private val context: Context) {
     }
 
     fun onResume() {
-        subscriptions = RxUtils.getNewCompositeSubIfUnsubscribed(subscriptions)
```

Comment:
This caused a lateinit exception. 

I may be wrong, but I don't see a reason to not just create a new sub here.